components/brave_welcome_ui/state/hooks.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import { BrowserType } from './component_types'
+import { BrowserProfile, ImportDataBrowserProxyImpl } from '../api/welcome_browser_proxy'
+import { loadTimeData } from '$web-common/loadTimeData'
+
+export const getValidBrowserProfiles = (profiles: BrowserProfile[]) => {
+  const browserList = Object.values(BrowserType)
```

Comment:
this variable can be defined at module-level instead of every time this function is called, since it's value is constant

Reply:
good call, done